RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplateReco.h

```diff
@@ -50,49 +53,52 @@
 #ifndef SiPixelTemplateReco_h
 #define SiPixelTemplateReco_h 1
 
-#include "RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplateDefs.h"
-
 #ifndef SI_PIXEL_TEMPLATE_STANDALONE
+#include "RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplateDefs.h"
 #include "RecoLocalTracker/SiPixelRecHits/interface/SiPixelTemplate.h"
 #else
+#include "SiPixelTemplateDefs.h"
 #include "SiPixelTemplate.h"
 #endif
 
 #define N2D 500
 
 #include <vector>
 
+#ifndef SiPixelTemplateClusMatrix
+#define SiPixelTemplateClusMatrix 1
```

Comment:
There is an include guard for this file already and ```SiPixelTemplateClusMatrix``` is not appearing anywhere else.
So, this ifndef has no effect in current CMSSW and looks more like a source of confusion